extensions/example-extension/renderer.tsx

```diff
@@ -9,13 +9,16 @@ export default class ExampleExtension extends LensRendererExtension {
       title: "Example Extension",
       components: {
         Page: () => <ExamplePage extension={this}/>,
-      }
+      },
+      params: [
+        exampleNameUrlParam,
+      ]
     }
   ];
 
   clusterPageMenus = [
     {
-      target: { pageId: "example", params: {} },
+      target: { pageId: "example" },
```

Comment:
And I thought this _was_ a good place to set specific param values. I'm confused.

Reply:
Yes, it's possible to define target params.
Side-note: page-menu-registry can specify **values**, page-registry consume those values with registered params.